example/src/main/java/org/apache/rocketmq/example/benchmark/BatchProducer.java

```diff
@@ -93,6 +96,19 @@ public static void main(String[] args) throws MQClientException {
         }
 
         final DefaultMQProducer producer = initInstance(namesrv, msgTraceEnable, rpcHook);
+
+        if (enableCompress) {
+            String compressType = commandLine.hasOption("ct") ? commandLine.getOptionValue("ct").trim() : "ZLIB";
+            int compressLevel = commandLine.hasOption("cl") ? Integer.parseInt(commandLine.getOptionValue("cl")) : 5;
+            int compressOverHowMuch = commandLine.hasOption("ch") ? Integer.parseInt(commandLine.getOptionValue("ch")) : 4;
```

Comment:
How about changing the default value of compressOverHowMuch to 4096?

Reply:
You are right. It's my carelessness.